mm2src/mm2_main/tests/docker_tests/sia_docker_tests.rs

```diff
@@ -0,0 +1,121 @@
+use mm2_number::MmNumber;
+use sia::http_client::{SiaApiClient, SiaApiClientError, SiaHttpConf};
+use sia::http_endpoints::{AddressBalanceRequest, AddressUtxosRequest, ConsensusTipRequest, TxpoolBroadcastRequest};
+use sia::spend_policy::SpendPolicy;
+use sia::transaction::{SiacoinOutput, V2TransactionBuilder};
+use sia::types::Address;
+use sia::{Keypair, PublicKey, SecretKey};
+use std::process::Command;
+use std::str::FromStr;
+use url::Url;
+
+#[cfg(test)]
+fn mine_blocks(n: u64, addr: &Address) {
+    Command::new("docker")
+        .arg("exec")
+        .arg("sia-docker")
+        .arg("walletd")
+        .arg("mine")
+        .arg(format!("-addr={}", addr))
+        .arg(format!("-n={}", n))
+        .status()
+        .expect("Failed to execute docker command");
+}
```

Comment:
I guess you use this to fill addresses for now since withdraw is not implemented yet, right?

Reply:
It is used for both funding addresses and progressing the chain in docker tests.